modules/gateway/gateway_test.go

```diff
@@ -0,0 +1,76 @@
+package gateway
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/consensus"
+	"github.com/NebulousLabs/Sia/modules"
+)
+
+var port = 9001
```

Comment:
haha me too, our port numbers are all over the place. I tried to fix it; see top comment.